shadowsocks-rust.yaml

```diff
@@ -30,6 +30,12 @@ pipeline:
       rustup override set stable
       rustup target add "${{host.triplet.rust}}"
 
+  # cargo bump has issues with packages if there are multiple versions of the same package.
+  - name: Update package with cargo
+    runs: |
+      cargo update -p idna@0.5.0
+      cargo update -p idna@1.0.2
```

Comment:
why do you update to 0.5.0 and then 1.0?2? Can't go straight to 1.0.2?

Reply:
Multiple different packages are using different versions of idna. Had to update one before the other one. Their was a 3rd version of idna but that go updated as part of the hashbrown update. This also isn't updating to 1.0.2 but 1.0.3.

Reply:
Thanks @kbsteere. This would be great to capture on our wiki as we build up some content for rust:
 - https://eng.inky.wtf/docs/squads/sustaining/playbooks/bumping_rust_dependencies/